src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -856,7 +856,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       <_FilesToBundle Include="@(_ResolvedFileToPublishPreserveNewest)" 
                       Condition="'%(_ResolvedFileToPublishPreserveNewest.ExcludeFromSingleFile)' != 'true'"/>
       <_FilesToBundle Include="@(_ResolvedFileToPublishAlways)" 
-                      Condition="'%(_ResolvedFileToPublishPreserveNewest.ExcludeFromSingleFile)' != 'true'"/>
+                      Condition="'%(_ResolvedFileToPublishAlways.ExcludeFromSingleFile)' != 'true'"/>
```

Comment:
why not just using `Condition="'%(ExcludeFromSingleFile)' != 'true'"`

Reply:
We cannot skip the `_ResolvedFileToPublishPreserveNewest`/`_ResolvedFileToPublishAlways` qualification here, because not all items in these groups define the `ExcludeFromSingleFile` property.